install_config/provisioners.adoc

```diff
@@ -15,35 +15,37 @@ toc::[]
 
 An external provisioner is an application that enables dynamic provisioning for
 a particular storage provider. External provisioners can run alongside the
-provisioner plug-ins provided by {product-title} and are configured in a
-similar way using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc[_StorageClasses_]. However, being external, they can be deployed
-and updated independently of {product-title} on an as-needed basis.
+provisioner plug-ins provided by {product-title} and are configured in a similar
+way as the *_StorageClass_* objects are configured, as described in the  xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#[Dynamic Provisioning and Creating Storage Classes] section.
+Since these provisioners are external, you can deploy and update them
+independently of {product-title}.
 
 [[provisioners-before-you-begin]]
 == Before You Begin
+This section provides information for preparing to use Ansible and
+includes information about adding parameters to the Ansible inventory file, to
+configure which external provisioners to install and their configuration. An link:https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/openshift_provisioners.yml[Ansible Playbook] is also available to deploy and upgrade external provisioners.
 
-An Ansible playbook is available to deploy and upgrade external provisioners. You
-should familiarize yourself with the
-xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation]
-section. This provides information for preparing to use Ansible and includes
-information about configuration. Parameters are added to the Ansible inventory
-file to configure which external provisioners to install and various areas of
-them.
+[NOTE]
+====
+Before proceeding, familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation] section.
+====
```

Comment:
@wongma7 Do we need to link to the complete topic? Or just a sub-topic (for example link just to the https://docs.openshift.com/container-platform/3.5/install_config/install/advanced_install.html#configuring-ansible section)